client/src/components/TabBody/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Textarea from '../Textarea'
+import './style.css'
+
+export default function TabBody(props) {
+
+  const { question, tips, expectations, onChange, wordsLimit } = props;
+
+  return (
+    <div className='tabBody'>
+      <div className='tabBody-guidelines'>
+        <h2>Refer to
+          <a className='tabBody-guidelines-href' href='https://esmeefairbairn.org.uk/userfiles/Documents/Application%20Forms/GuidanceforGrantApplicants.pdf'>
+            application guidelines
+          </a>
+        </h2>
+      </div>
+      <h1 className='tabBody-question'>{question}</h1>
+      <ul>
+        {tips.map(item => <li>{item}</li>)}
+      </ul>
+      <h2 className='tabBody-answer'>We do not fund:</h2>
```

Comment:
The class name is not correct!! 
this is not an answer

Reply:
Edited